File: src/main/resources/output/IFC4X3_csharp/Entities/IfcOrganization/IfcOrganization.cs
public class IfcOrganization
{
    public IfcIdentifier Identification { get; set; }
    public IfcLabel Name { get; set; }
    public IfcText Description { get; set; }
    public List<IfcActorRole> Roles { get; set; }
    public List<IfcAddress> Addresses { get; set; }

    // === INVERSE CLAUSES ===
    // IsRelatedBy : SET [0:?] OF IfcOrganizationRelationship FOR RelatedOrganizations
    // Relates : SET [0:?] OF IfcOrganizationRelationship FOR RelatingOrganization
    // Engages : SET [0:?] OF IfcPersonAndOrganization FOR TheOrganization
}

File: src/main/resources/output/IFC4X3_js/Entities/IfcOrganization/IfcOrganization.js
class IfcOrganization {
    constructor() {
        /** @type {IFCIDENTIFIER} */
        this.Identification = null;
        /** @type {IFCLABEL} */
        this.Name = null;
        /** @type {IFCTEXT} */
        this.Description = null;
        /** @type {IFCACTORROLE[]} */
        this.Roles = null;
        /** @type {IFCADDRESS[]} */
        this.Addresses = null;
    }

    // === INVERSE CLAUSES ===
    // IsRelatedBy : SET [0:?] OF IfcOrganizationRelationship FOR RelatedOrganizations
    // Relates : SET [0:?] OF IfcOrganizationRelationship FOR RelatingOrganization
    // Engages : SET [0:?] OF IfcPersonAndOrganization FOR TheOrganization
}
